presto-main/src/main/java/com/facebook/presto/server/ServerMainModule.java

```diff
@@ -350,12 +353,13 @@ protected void setup(Binder binder)
 
         // statistics calculator
         binder.bind(CostComparator.class).in(Scopes.SINGLETON);
-        binder.bind(StatsCalculator.class).to(CoefficientBasedStatsCalculator.class).in(Scopes.SINGLETON);
         binder.bind(CostCalculator.class).to(CostCalculatorUsingExchanges.class).in(Scopes.SINGLETON);
         binder.bind(CostCalculator.class)
                 .annotatedWith(EstimatedExchanges.class)
                 .to(CostCalculatorWithEstimatedExchanges.class).in(Scopes.SINGLETON);
         binder.bind(Lookup.class).to(StatelessLookup.class).in(Scopes.SINGLETON);
+        binder.bind(StatsCalculator.class).annotatedWith(SelectingStatsCalculator.Old.class).to(CoefficientBasedStatsCalculator.class).in(Scopes.SINGLETON);
```

Comment:
In practice I'm not sure the CoefficientBasedStatsCalculator remains useful because it doesn't deal with symbol statistics, and so the CostCalculators can't get a cost.

you could:
1) remove it entirely and only use the new calculator
2) update it to compute symbol stats
3) change the CostCalculators to be able to do something useful with only row count (maybe valuable anyway for connectors that can only provide minimal statistics)